ops/ansible/playbooks-ccs/files/client_data_server_ab2d_sbx_certificate.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
+MIIEsDCCApgCCQD2KLEEIwKWMDANBgkqhkiG9w0BAQsFADAaMRgwFgYDVQQDDA9h
+YjJkLXNieC1jbGllbnQwHhcNMjAwMTIzMTQ0NzIzWhcNMjUwMTIxMTQ0NzIzWjAa
+MRgwFgYDVQQDDA9hYjJkLXNieC1jbGllbnQwggIiMA0GCSqGSIb3DQEBAQUAA4IC
+DwAwggIKAoICAQDaY8UJw58igvUHVOFD7l8cu//yqaN/i2exIXTu5UipAa+7MmC5
+YkyWGMgThLeO113urOVbrsuwR375JqlRIvqo6aRiS6UAoJgyOdK7UZ0d1oSzwhsJ
+x0DfYqqdoxcGeO7aR++Z4R2riqo4S3nIOaaIg+Xzx/SAlCCFGAdHQSSYthFe3EWG
+hVnmByA9FFpuD1hGM9RPId9dJX4z+URlUQuwNq8QMRZG/FK9rc8yUuwDHP5oxBhr
+jrFt5m6x7RBNoMz9zkXb6EehG+b394eIdb+sw07drjXGhhFWgw+rlNVAtzPLIKf9
+zBjWV9JcglTC9o8JKXm3Np6qrckdPj+lLiWbw9MEHyO5IOWUfkY5IlKebYaRCnUI
+EMEnZLDd1wxa2qLQ46761dqva6UXOkrArVG528nJR7sHzV+xTNUS5DcPHh8/g3f2
+gt8xTNx8DutHGFyEyqDr/Ijphd+gN5b8KZypVH6HC53lfsp7aKA2YLqY3jVp0xzU
+xpyH9dipTtd9qh50KL21CA4V7DKOX2rcOOMfMDVZzQ735Hf8w91bCDwOf2tkPEu8
+coinXkIZniAKhDf8s9y4uA+npgXwCfudHf2F+JnNk82iJEmNFMRCWeqD92P8MraR
+uLO6V0FzmACRs+hwebo/pT3PUVq1EIoQocPzz053wV1C6kZoTkNKFbf0uwIDAQAB
+MA0GCSqGSIb3DQEBCwUAA4ICAQCWcTn+DENk0LYuR7sxj/QRh9VyZxUBy+IZqtFN
+kMQ2WACdANbQtXT/VkvKYOTUAxhEy8fhpUZShGNOPHJj3oo9ufNe6cmRbpum55gj
+f6dR7y1hsZdOY8tchXLUs1Q58av7pAUMkb14cnYXvPrvh5jVoAoRtU2ECAo9+EvJ
+55H79nJd5VU9jiEazF4T+lvCqOEn+b4NoaI+3t6dfnfjsF0uvdw4dKQ3KR0eNNNE
+zUdC/tOi1BHlWBXZg1Mz246p+tGilHI91fQ334MLLY56Ed5GiJOl3/T++/GrFEpp
+Vxx1ZLxrbceBZB7LC6sJVRfZSf3BM3xo06GrmL1nGJQNR+0XEMDIqGYbt6iAuPqd
+WSRiP6txnlrLLhZK/SytOnAwBYE0kKupuFpYH0QXuBn1SR4VqSi78pGb4otfSydu
+KBZW8d5YRPzrBvUUzHzwr26x7V+8mDMYzGVzPrbzm1pztj3wjGu5SyPfPsD7zNHV
+NhpgXk/Q80Pevcu9nR1l1L44VlQQCTCi9VttkQHuPwOUWEeZ2Hn3y259+/EGvlzr
+ulaoFugpy9q4EaNqZzQmh9sOYI9Vk6SKHwQvlOhQt9gRBNjLA6j+7vkWLUbVCFZx
+MdoTmtkGSzp1/1TcIhcGxwUqyk/jFKYvsoTsIWaUmdGb50QyHBIgfPj2AU1xaARb
+67UkIw==
+-----END CERTIFICATE-----
```

Comment:
```
openssl x509 -text -in ops/ansible/playbooks-ccs/files/client_data_server_ab2d_sbx_certificate.pem
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 17737621763724056112 (0xf628b10423029630)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=ab2d-sbx-client
        Validity
            Not Before: Jan 23 14:47:23 2020 GMT
            Not After : Jan 21 14:47:23 2025 GMT
        Subject: CN=ab2d-sbx-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:da:63:c5:09:c3:9f:22:82:f5:07:54:e1:43:ee:
                    5f:1c:bb:ff:f2:a9:a3:7f:8b:67:b1:21:74:ee:e5:
                    48:a9:01:af:bb:32:60:b9:62:4c:96:18:c8:13:84:
                    b7:8e:d7:5d:ee:ac:e5:5b:ae:cb:b0:47:7e:f9:26:
                    a9:51:22:fa:a8:e9:a4:62:4b:a5:00:a0:98:32:39:
                    d2:bb:51:9d:1d:d6:84:b3:c2:1b:09:c7:40:df:62:
                    aa:9d:a3:17:06:78:ee:da:47:ef:99:e1:1d:ab:8a:
                    aa:38:4b:79:c8:39:a6:88:83:e5:f3:c7:f4:80:94:
                    20:85:18:07:47:41:24:98:b6:11:5e:dc:45:86:85:
                    59:e6:07:20:3d:14:5a:6e:0f:58:46:33:d4:4f:21:
                    df:5d:25:7e:33:f9:44:65:51:0b:b0:36:af:10:31:
                    16:46:fc:52:bd:ad:cf:32:52:ec:03:1c:fe:68:c4:
                    18:6b:8e:b1:6d:e6:6e:b1:ed:10:4d:a0:cc:fd:ce:
                    45:db:e8:47:a1:1b:e6:f7:f7:87:88:75:bf:ac:c3:
                    4e:dd:ae:35:c6:86:11:56:83:0f:ab:94:d5:40:b7:
                    33:cb:20:a7:fd:cc:18:d6:57:d2:5c:82:54:c2:f6:
                    8f:09:29:79:b7:36:9e:aa:ad:c9:1d:3e:3f:a5:2e:
                    25:9b:c3:d3:04:1f:23:b9:20:e5:94:7e:46:39:22:
                    52:9e:6d:86:91:0a:75:08:10:c1:27:64:b0:dd:d7:
                    0c:5a:da:a2:d0:e3:ae:fa:d5:da:af:6b:a5:17:3a:
                    4a:c0:ad:51:b9:db:c9:c9:47:bb:07:cd:5f:b1:4c:
                    d5:12:e4:37:0f:1e:1f:3f:83:77:f6:82:df:31:4c:
                    dc:7c:0e:eb:47:18:5c:84:ca:a0:eb:fc:88:e9:85:
                    df:a0:37:96:fc:29:9c:a9:54:7e:87:0b:9d:e5:7e:
                    ca:7b:68:a0:36:60:ba:98:de:35:69:d3:1c:d4:c6:
                    9c:87:f5:d8:a9:4e:d7:7d:aa:1e:74:28:bd:b5:08:
                    0e:15:ec:32:8e:5f:6a:dc:38:e3:1f:30:35:59:cd:
                    0e:f7:e4:77:fc:c3:dd:5b:08:3c:0e:7f:6b:64:3c:
                    4b:bc:72:88:a7:5e:42:19:9e:20:0a:84:37:fc:b3:
                    dc:b8:b8:0f:a7:a6:05:f0:09:fb:9d:1d:fd:85:f8:
                    99:cd:93:cd:a2:24:49:8d:14:c4:42:59:ea:83:f7:
                    63:fc:32:b6:91:b8:b3:ba:57:41:73:98:00:91:b3:
                    e8:70:79:ba:3f:a5:3d:cf:51:5a:b5:10:8a:10:a1:
                    c3:f3:cf:4e:77:c1:5d:42:ea:46:68:4e:43:4a:15:
                    b7:f4:bb
                Exponent: 65537 (0x10001)
    Signature Algorithm: sha256WithRSAEncryption
```
There's no private key material here, right? ✅
Is the cert different from any other certs from the same project (just watching out for oopses)? ✅
Is its Subject unique? ✅
Does it have a reasonably useful/informative Subject? ✅
Does it have a reasonable key size (>= 2048)? ✅